include/cantera/thermo/PengRobinson.h

```diff
@@ -0,0 +1,359 @@
+//! @file PengRobinson.h
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_PENGROBINSON_H
+#define CT_PENGROBINSON_H
+
+#include "MixtureFugacityTP.h"
+#include "cantera/base/Array.h"
+
+namespace Cantera
+{
+/**
+ * Implementation of a multi-species Peng-Robinson equation of state
+ *
+ * @ingroup thermoprops
+ */
+class PengRobinson : public MixtureFugacityTP
+{
+public:
+    //! @name Constructors and Duplicators
+    //! @{
+
+    //! Base constructor.
+    PengRobinson();
+
+    //! Construct and initialize a PengRobinson object directly from an
+    //! ASCII input file

```

Comment:
I don't think the input files are required to be ASCII... it would depend on how the `yaml-cpp` library handles Unicode. In any case, I don't think there's a need to be so specific here. Just "from an input file" should be fine.

Reply:
It was copied from Redlich-Kwong file. Nevethless, I have removed 'ASCII' from both the files.